### PR TITLE
At os family setup

### LINKED ISSUE #148: Extend AT creation with osfamily setting

### PR DESCRIPTION
**Pull request by Nuanda**
Fixes #148

### CODE REVIEW

**Comment by Nuanda** [comment]
@tbartynski Tomek, would you review that PR. This is needed to fix the problem we were discussing yesterday.

File: spec/providers/atmosphere/optimization_strategy/default_spec.rb [hunk 1]
```diff
@@ -37,21 +37,27 @@ def appliance(options)
...
-  let!(:tmpl_of_shareable_at) { create(:virtual_machine_template, appliance_type: shareable_appl_type, compute_site: openstack)}
-
   context 'new appliance created' do
+    let(:wf) { create(:workflow_appliance_set) }
+    let(:openstack) { create(:openstack_with_flavors, funds: [fund]) }
+    let(:fund) { create(:fund) }
+    let(:shareable_appl_type) { create(:shareable_appliance_type) }
+
+    before do
+      # os family reused by AT, Flavors - simply magic :(
```
Comment (nowakowski) [inline spec/providers/atmosphere/optimization_strategy/default_spec.rb]: It's not magic; `openstack_with_funds` is an OpenStack-type CS with 5 flavors created automatically.

**Comment by nowakowski** [comment]
:+1:

**Comment by Nuanda** [comment]
DO NOT REVIEW, still under investigation.

**Comment by Nuanda** [comment]
@mkasztelnik Marek, check my latest commit with different travis setup - is this ok? Also, take a look on your acbb9e6 commit and see if this is still needed.

**Comment by mkasztelnik** [comment]
@Nuanda it looks ok, local and travis tests are green :smile: I added 3 small changes: make sure that `OsFamilies` are created only once (3b8d205be0ce077ddd721e480b94b5473e31ee12), restore creating admin user in `development` and `production` (bce7cfe912da81cd67cbaba668fc4014d5e32e3c - i'm using it a lot while developing Atmosphere) and last one is a little cleanup (e64fa51b267c2cd881566dfcfab2ffc99e1f7a4e).

The only thing we can improve is to change default rake task to something different.

IMHO this is ready for review - @tbartynski can you take a look?

**Comment by nowakowski** [comment]
Still looks good - I sustain my :+1:

**Comment by tbartynski** [comment]
:+1:

**Pull request merged**